### ISSUE TITLE
Docs are incorrect

### ISSUE THREAD

**Issue by stramel** [luisrudge/postcss-flexbugs-fixes#42]
Input

```css
.foo { flex: 1; }
.bar { flex: 1 1; }
.foz { flex: 1 1 0; }
.baz { flex: 1 1 0px; }
```

**No Longer** Outputs

```css
.foo { flex: 1 1 0%; }
.bar { flex: 1 1 0%; }
.foz { flex: 1 1 0%; }
.baz { flex: 1 1 0%; }
```

instead it outputs:

```css
.foo { flex: 1 1; }
.bar { flex: 1 1; }
.foz { flex: 1 1; }
.baz { flex: 1 1; }
```

**Comment by malectro** [comment]
looks like it was this commit https://github.com/luisrudge/postcss-flexbugs-fixes/commit/bd68bc11531fb63b3ecb448345e41e181a1b4e8b

it's unclear to me why this was added (even given the Safari issue). totally changed the behavior of the package.

**Comment by luisrudge** [comment]
How do you propose we fix this? rollback that commit even though it breaks on safari?

**Comment by malectro** [comment]
is this still in an issue in Safari 11? it seems to be parsing `0%` just fine in my application, but i'm struggling to find anything online showing how to repro the it. there's no codepen or anything showing the issue here https://github.com/luisrudge/postcss-flexbugs-fixes/pull/38 as far as i can tell.

i'd recommend just reverting the offending code https://github.com/luisrudge/postcss-flexbugs-fixes/compare/master...Spaced-Out:basis-default?expand=1 and keeping the most recent addition. if the Safari bug is still present, then a major semver bump would at least be merited.

**Comment by luisrudge** [comment]
Thanks @malectro for all the help! I released a new version: `postcss-flexbugs-fixes@3.3.0`

btw, would you mind if I add you as a maintainer of this repo?
